Controller/Admin/TransactionTab.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+namespace Wirecard\Oxid\Controller\Admin;
+
+use Wirecard\Oxid\Model\Transaction;
+
+use OxidEsales\Eshop\Core\Registry;
+use OxidEsales\Eshop\Application\Controller\Admin\AdminDetailsController;
+use OxidEsales\Eshop\Application\Model\Order;
+use OxidEsales\Eshop\Application\Model\Payment;
+
+/**
+ * Controls the view for a single transaction tab.
+ */
+class TransactionTab extends AdminDetailsController
+{
+    protected $_sThisTemplate = 'transaction_tab.tpl';
+
+    protected $_i18n = null;
+
+    /**
+     * TransactionTab constructor.
+     */
+    public function __construct()
+    {
+        parent::__construct();
+        
+        $this->_i18n = Registry::getLang();
+    }
+
+    /**
+     * @inheritdoc
+     *
+     * @return string
+     */
+    public function render(): string
+    {
+        $objectId = $this->getEditObjectId();
+
+        if (isset($objectId) && $objectId !== '-1') {
+            $transaction = oxNew(Transaction::class);
+            $transaction->load($objectId);
+            $order = oxNew(Order::class);
+            $order->load($transaction->wdoxidee_ordertransactions__wdoxidee_orderid->value);
+            $payment = oxNew(Payment::class);
+            $payment->load($order->oxorder__oxpaymenttype->value);
+
+            $this->_aViewData['listData'] = $this->getListData($transaction, $order, $payment);
+        } else {
+            $this->_aViewData['listData'] = [];
+        }
+
+        $this->_aViewData['controller'] = $this->classKey;
+
+        return parent::render();
+    }
+
+    /**
+     * Returns an array of data used to populate the view.
+     *
+     * @param Transaction $transaction
+     * @param Order       $order
+     * @param Payment     $payment
+     * @return array
+     */
+    public function getListData(Transaction $transaction, Order $order, Payment $payment): array
+    {
+        return [];
```

Comment:
Please, add a **TODO comment** for every implementation detail that is still missing.

Reply:
No need for a TODO here, this is the base class and an empty array is valid.